test/dotcom/system_status/groups_test.exs

```diff
@@ -5,177 +5,182 @@ defmodule Dotcom.SystemStatus.GroupsTest do
+                     informed_entity: [
+                       InformedEntity.build(:informed_entity, route: affected_route_id)
+                     ],
+                     active_period: [{alert_start_time, alert_end_time}]
                    )
                  ],
-                 now()
+                 time
                )
+               |> statuses_for(affected_route_id)
+    end
 
-      assert [
-               %{
-                 route_id: "Blue",
-                 sub_routes: [],
-                 statuses: [%{description: "Shuttle Buses", time: nil}]
-               },
-               %{
-                 route_id: "Orange",
-                 sub_routes: [],
-                 statuses: [%{description: "Normal Service", time: nil}]
-               },
-               %{
-                 route_id: "Red",
-                 sub_routes: [],
-                 statuses: [%{description: "Normal Service", time: nil}]
-               },
-               %{
-                 route_id: "Green",
-                 sub_routes: [],
-                 statuses: [%{description: "Normal Service", time: nil}]
-               }
-             ] =
-               Groups.groups(
-                 [
-                   Alert.build(:alert,
-                     effect: :shuttle,
-                     informed_entity: [InformedEntity.build(:informed_entity, route: "Blue")],
-                     active_period: [current_active_period()]
-                   )
-                 ],
-                 now()
-               )
+    test "when there's an alert for a heavy rail line, shows 'Normal Service' for the other lines" do
+      affected_route_id = Faker.Util.pick(@heavy_rail_lines)
+
+      time = time_today()
+      alert_start_time = time_before(time)
+      alert_end_time = time_after(time)
+
+      groups =
+        Groups.groups(
+          [
+            Alert.build(:alert,
+              effect: :delay,
+              informed_entity: [InformedEntity.build(:informed_entity, route: affected_route_id)],
+              active_period: [{alert_start_time, alert_end_time}]
+            )
+          ],
+          time
+        )
+
+      @heavy_rail_lines
+      |> List.delete(affected_route_id)
+      |> Enum.each(fn route_id ->
+        assert [
+                 %{
+                   statuses: [%{time: nil, description: "Normal Service"}],
+                   sub_routes: [],
+                   route_id: ^route_id
+                 }
+               ] = groups |> statuses_for(route_id)
+      end)
     end
 
     test "shows future active time for alerts that will become active later in the day" do
+      affected_route_id = Faker.Util.pick(@heavy_rail_lines)
+
+      time = time_today()
+      alert_start_time = time_after(time)
+      alert_end_time = time_after(alert_start_time)
+
+      alert_start_time_display =
+        alert_start_time |> Timex.format!("{kitchen}") |> String.downcase()
```

Comment:
Shouldn't this use the Util function that does this?

Reply:
Done! (When I first did this work, that Util function hadn't been merged yet)